src/web/app/pages-monitoring/logs-page/logs-page.component.scss

```diff
@@ -21,15 +21,15 @@
 }
 
 @media (max-width: 768px) {
-  .col-6 {
+  .col-4 {
     width: auto;
   }
 
-  .col-4 {
+  .col-1 {
     width: auto;
   }
 
-  .col-1 {
+  .col-6 {
     width: auto;
   }
```

Comment:
I think let's remove these lines so as not to override the Bootstrap default styles as the overriding might cause issues in the future. Sorry for any confusion caused! We must have missed these out when removing the other Bootstrap style overrides.